slices/zstd.yaml

```diff
@@ -0,0 +1,31 @@
+package: zstd
+
+essential:
+  - zstd_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - liblz4-1_libs
+      - liblzma5_libs
+      - libstdc++6_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/pzstd:
+      /usr/bin/unzstd: {symlink: /usr/bin/zstd}
+      /usr/bin/zstd:
+      /usr/bin/zstdcat: {symlink: /usr/bin/zstd}
+      /usr/bin/zstdmt: {symlink: /usr/bin/zstd}
```

Comment:
Another question we have: we see in some other slice definitions that some files that are symlinks are just listed directly in the contents without the `symlink:` attribute being marked in the SDF.

For example, in xz-utils.yaml, /usr/bin/unxz, /usr/bin/xzcat, /usr/bin/xzcmp, /usr/bin/xzegrep & /usr/bin/xzfgrep are all symlinks pointing to either xz, xzdiff or xzgrep, but are not marked with the `symlink:` attribute in the SDF.

Should we follow suit here and drop the `symlink:` attribute in this SDF for zstd for /usr/bin/unzstd, /usr/bin/zstdcat & /usr/bin/zstdmt?

Reply:
Yes, you can drop `symlink` when the file is already a `symlink` in the deb.

In short:
 - the deb contents will be extracted while preserving their type, mode and permissions,
 - `symlink` can be used to explicitly create a symlink that doesn't exist in the deb's contents (this is useful for reproducing the outcomes of maintainer scripts)

Reply:
Hi @cjdcordeiro,

Great, thank you for clarifying! We'll drop the `symlink:` attribute here in this case.